pretix_mollie/payment.py

```diff
@@ -116,13 +122,24 @@ def settings_form_fields(self):
             else:
                 return {}
         else:
+            api_key_kw = {}
+            if self.settings.api_key:
+                api_key_kw['required'] = False
+                # We can show the first 5 characters of the key, as it indicates test or live key.
+                api_key_kw['help_text'] = (_("A secret key has currently been configured: %s. The remainder of the "
+                                             "secret key has been hidden for security purposes.") %
+                                           (self.settings.api_key[:5] + "*" * len(self.settings.api_key[5:])))
             fields = [
                 ('api_key',
                  forms.CharField(
                      label=_('Secret key'),
                      validators=(
                          MollieKeyValidator(['live_', 'test_']),
                      ),
+                     widget=SecreyKeyInput(attrs={
+                         'autocomplete': 'new-password'  # see https://bugs.chromium.org/p/chromium/issues/detail?id=370363#c7
```

Comment:
I think we can just set this by default in SecretKeyInput, as I plan to re-use that field for Stripe, PayPal, etc.